### PR TITLE
[IMP] accounting: updated chart of accounts

### PR DESCRIPTION
**Pull request by toaa-odoo**
Creating account groups part was outdated.

task id: 2950820

### CODE REVIEW

**Review by chiaraprattico — APPROVED** [review]

**Comment by toaa-odoo** [comment]
@jcs-odoo kind reminder if you have time

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hello @toaa-odoo and @chiaraprattico (Chiara, pay attention to the comment about the dev mode, it's important to check this as a reviewer too).

since you're updating the page, here are some elements that could be updated as well.

- replace `*` list marks with `-` instead
- rename old image files (hyphens instead of underscores and no numbering)

Also, I think these changes can be applied from 14.0, even if your new screenshot isn't exactly the same, that's okay :)

**Comment by toaa-odoo** [comment]
@jcs-odoo It's the PR with which we had issues and had to turn into a draft, and force-push, and then make it ready. I don't think it was merged yet ?

File: content/applications/finance/accounting/getting_started/initial_configuration/chart_of_accounts.rst [hunk 1]
```diff
@@ -129,43 +126,56 @@ Tags
...
 --------------------
 
 Some accounts, such as accounts made to record the transactions of a payment method, can be used for
 the reconciliation of journal entries.
 
-For example, an invoice paid with a credit card can be *marked as paid* if reconciled with the
-payment. Therefore, the account used to record credit card payments needs to be configured as
-*allowing reconciliation*.
+For example, an invoice paid with a credit card can be marked as :guilabel:`paid` if reconciled with
+the payment. Therefore, the account used to record credit card payments needs to be configured as
```
[the remark this answers is not in the document]
Comment (toaa-odoo) [reply content/applications/finance/accounting/getting_started/initial_configuration/chart_of_accounts.rst:167]: Yes, you should work in accounting

**Comment by toaa-odoo** [comment]
@xpl-odoo Could you have a (final) look ? :)

File: content/applications/finance/accounting/getting_started/initial_configuration/chart_of_accounts.rst [hunk 2]
```diff
@@ -129,46 +126,60 @@ Tags
...
-**Account Groups** are useful to list multiple accounts as *sub-accounts* of a bigger account and
-thus consolidate reports such as the **Trial Balance**.
+**Account groups** are useful to list multiple accounts as *sub-accounts* of a bigger account and
+thus consolidate reports such as the **Trial Balance**. By default, groups are handled automatically
+based on the code of the group. For example, a new account `131200` is going to be part of the group
+`131000`.
+
+.. note::
+   Regular users should not need to create account groups manually. The following section is only
+   intended for rare and advanced use cases.
```
[the remark this answers is not in the document]
Comment (toaa-odoo) [reply content/applications/finance/accounting/getting_started/initial_configuration/chart_of_accounts.rst:142]: Done! Pinging doc-review

File: content/applications/finance/accounting/getting_started/initial_configuration/chart_of_accounts.rst [hunk 3]
```diff
@@ -129,46 +126,60 @@ Tags
...
+
+.. image:: chart_of_accounts/account-groups.png
+   :align: center
+   :alt: Account groups creation.
 
-To display your **Trial Balance** report with your Account Groups, go to :menuselection:`Accounting
---> Reporting --> Trial Balance`, then open the *Options* menu and select **Hierarchy and
-Subtotals**.
+To display your **Trial Balance** report with your account groups, go to :menuselection:`Accounting
+app-->Reporting-->Trial Balance`, then open the :guilabel:`Options` menu and select
```
Comment (AntoineVDV) [inline content/applications/finance/accounting/getting_started/initial_configuration/chart_of_accounts.rst:154]: Broken markup
image

```suggestion
app --> Reporting --> Trial Balance`, then open the :guilabel:`Options` menu and select
```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

File: content/applications/finance/accounting/getting_started/initial_configuration/chart_of_accounts.rst (hunk 3 above)
Comment (toaa-odoo) [reply content/applications/finance/accounting/getting_started/initial_configuration/chart_of_accounts.rst:155]: Fixed

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**